acceptance/testdata/launcher/exec.d/fd_unix.go

```diff
@@ -0,0 +1,9 @@
+//+build linux darwin
```

Comment:
Should we call this file `fd_linux` instead of `fd_unix`? I'm not sure what is more accurate.

Reply:
In other places where we have _windows.go files we have a _unix.go counterpart. Since it will also compile on Mac I think _unix.go is more accurate.